### ISSUE TITLE
data-source/cloudsigma_vlan: add possibility to filter by name

### ISSUE THREAD

**Issue by pavel-github** [cloudsigma/terraform-provider-cloudsigma#14]
# Feature request

### Description

We want to have more filter possibilities for data source VLAN (e.g. `name`, `tags`).
At the moment this attributes are nested in JSON VLAN structure so it's not possible to use it.

### New or Affected Resource(s)

* cloudsigma_vlan

### Potential Terraform Configuration

```hcl
# Copy-paste your Terraform configurations here - for large Terraform configs,
# please use a service like Dropbox and share a link to the ZIP file. For
# security, you can also encrypt the files using our GPG public key.
```

### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/)
  to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

**Comment by pavel-github** [comment]
**tl;dr:** how to filter VLANs by name or UUID from v2.4.0+ version

```terraform
data "cloudsigma_vlan" "my_vlan" {
  name = "bastion vlan"
}
...
data "cloudsigma_vlan" "my_vlan" {
  uuid = "10619300-edda-42ba-91e0-7e3df0689d00"
}
```
